lib/application.js

```diff
@@ -105,9 +105,9 @@ module.exports = class Application extends Emitter {
   use(fn) {
     if (typeof fn !== 'function') throw new TypeError('middleware must be a function!');
     if (isGeneratorFunction(fn)) {
-      deprecate('Support for generators will been removed in v3. ' +
-                'See the documentation for examples of how to convert old middleware ' +
-                'https://github.com/koajs/koa/tree/v2.x#old-signature-middleware-v1x');
+      deprecate(`Support for generators will been removed in v3. 
```

Comment:
the intention before was that there was no indentation or line break before. now there is. do you guys think this is uglier?